### DISCUSSION TITLE
Welcome to 3270font Discussions!

### DISCUSSION THREAD

**Discussion by rbanffy** [rbanffy/3270font#93] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by spidey3** [comment]
Hi - is there any place that hosts a .css file for these fonts, so that they can be used on a web page?

**Reply by rbanffy** [reply]
I'd say the easy way would be to download the zip file and host the CSS along with the font binary - the license allows that. For now, I also recommend using the TTF version for this, as the others don't look great.
